scrapers/Clips4Sale.yml

```
@@ -1,45 +1,87 @@
-name: "Clips4Sale"
+name: Clips4Sale
+
 sceneByURL:
   - action: scrapeXPath
     url:
-      - clips4sale.com
-    scraper: sceneScraper
+      - clips4sale.com/studio/
+    scraper: c4sSceneScraper
+  - action: scrapeXPath
+    url:
+      - clips4sale.com/search/clip/
+    scraper: c4sClipsSceneScraper
+
 xPathScrapers:
-  sceneScraper:
+  c4sSceneScraper:
     common:
       $studio: //span[contains(text(),"From:")]/following-sibling::a
     scene:
-      Title: //div[@data-clipid]/following-sibling::h3
+      Title:
+        selector: //div[@data-clipid]/following-sibling::h3
+        postProcess: &ppStripTitle
+          - replace:
+              # https://regex101.com/r/CYLeuO/8
+              - regex: (?i)[ \t]*((Super )?[SH]D)?[ ,-]*(\b(MP4|OPTIMUM|WMV|MOV|AVI|UHD|[48]K)\b|1080p|720p|480p|(Standard|High) Def(inition)?)+[ \t]*
+                with:
       Details:
```

Sorry, just thought about it, might as well add it.
```suggestion
                with:
              # https://regex101.com/r/C3TgFG/1
              - regex: [ \t]*[\[(][\])][ \t]*|[ -]+$
                with:
      Details:
```

(and so nice you only have to add it once 😄)

---

oh, wait it's broken